.nvmrc

```diff
@@ -0,0 +1 @@
+11
```

Comment:
Why this, i thought the current best practice is requiring the node version on the package.json.

Reply:
This doesn't seem to be supported by nvm yet: https://github.com/nvm-sh/nvm/issues/651

Reply:
Yep, currently, it is better the have the `.nvmrc` file in a repo